amespahdbpythonsuite/fitted.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, d=None, **keywords):
         Spectrum.__init__(self, d, **keywords)
         self.weights = None
         self.observation = None
+        self.atoms = None
```

Comment:
This could probably be self.__atoms, as it won't be accessed out side the class.

Reply:
Given my previous comment, since the atoms variable is not present in any other module currently and thus there is no potential confusion with the attributes name, a single underscore should be sufficient to indicate it is a private variable for this class.